view/js/filebrowser.js

```diff
@@ -47,98 +47,115 @@
  *		$("body").on("fbrowser.image.example", function(event, filename, bbcode, id) {
  *			// close colorbox
  *			$.colorbox.close();
- *			// replace textxarea text with bbcode
+ *			// replace textarea text with bbcode
  *			$(id).value = bbcode;
  *		});
  **/
+const FileBrowser = {
+	nickname: '',
+	type: '',
+	event: '',
+	id: null,
 
-var FileBrowser = {
-	nickname : "",
-	type : "",
-	event: "",
-	id : null,
-
-	init: function(nickname, type) {
+	init: function (nickname, type) {
 		FileBrowser.nickname = nickname;
 		FileBrowser.type = type;
-		FileBrowser.event = "fbrowser."+type;
-		if (location['hash']!=="") {
-			var h = location['hash'].replace("#","");
+		FileBrowser.event = "fbrowser." + type;
+		if (location['hash'] !== "") {
+			const h = location['hash'].replace('#', '');
 			FileBrowser.event = FileBrowser.event + "." + h.split("-")[0];
 			FileBrowser.id = h.split("-")[1];
 		}
```

Comment:
While you're at it 😋
```suggestion
		FileBrowser.event = 'fbrowser.' + type;
		if (location['hash'] !== '') {
			const h = location['hash'].replace('#', '');
			FileBrowser.event = FileBrowser.event + '.' + h.split('-')[0];
			FileBrowser.id = h.split('-')[1];
		}
```